frontend/src/components/pages/homepage/Homepage.js

```diff
@@ -19,7 +21,7 @@ function Homepage() {
                autoplay
                loop
                src={WineAnimation}
-               style={{ height: "200px", width: "300px" }}>
+               style={{ height: "200px", width: "300px" ,padding:"0px 50px 0px 50px"}}>
```

Comment:
if you only give 2 values, the first one is for `top` and `bottom`, the second for `left` and `right`

```suggestion
               style={{ height: "200px", width: "300px" ,padding:"0px 50px"}}>
```